core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -3795,7 +3786,7 @@ protected function treeView()
 		{
 			for ($i=0, $c=\count($topMostRootIds); $i<$c; $i++)
 			{
-				$tree .= $this->generateTree($table, $topMostRootIds[$i], array('p'=>($topMostRootIds[($i-1)] ?? null), 'n'=>($topMostRootIds[($i+1)] ?? null)), $blnHasSorting, -20, ($blnClipboard ? $arrClipboard : false), (($GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['mode'] ?? null) == self::MODE_TREE && $blnClipboard && $topMostRootIds[$i] == $arrClipboard['id']), false, false, $arrFound);
+				$tree .= $this->generateTree($table, $topMostRootIds[$i], array('p'=>($topMostRootIds[($i-1)] ?? null), 'n'=>($topMostRootIds[($i+1)] ?? null)), true, -20, ($blnClipboard ? $arrClipboard : false), (($GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['mode'] ?? null) == self::MODE_TREE && $blnClipboard && $topMostRootIds[$i] == $arrClipboard['id']), false, false, $arrFound);
```

Comment:
Why did you remove `$blnHasSorting` in line 3725 and set it to `true` here? Not every tree view table has a `sorting` column.

Reply:
yes it has. It is enforced in https://github.com/contao/contao/blob/4.13/core-bundle/src/Resources/contao/drivers/DC_Table.php#L3669-L3673

Reply:
I looks like however we could have an _extended tree_ without sorting? Has anyone ever tried or used that?

Reply:
As discussed on the Contao Call yesterday I reverted the `$blnHasSorting` removal in 34c245813989e7a117ebbc15543402d84768c9e4

Reply:
As discussed on the Contao Call yesterday I reverted the `$blnHasSorting` removal in 34c245813989e7a117ebbc15543402d84768c9e4